### PR TITLE
Implement get UTXOs function

### PR DESCRIPTION
**Pull request by TrustHenry**
We need a function that returns the UTXOs for a given PublicKey.
It requests Stoa for public key's UTXOs and returns unspent UTXOs.

Related to #51

### CODE REVIEW

File: src/modules/net/BOAClient.ts [hunk 1]
```diff
@@ -17,11 +17,13 @@
 import { DataPayload } from '../data/DataPayload';
 import { Hash, hash, hashFull } from '../data/Hash';
 import { Request } from './Request';
+import { UnspentTxOutput } from './response/UnspentTxOutput';
 import { Validator } from './response/Validator';
 import { Seed, Transaction, TxInput, TxOutput } from "../..";
 
 import { AxiosResponse, AxiosError } from 'axios';
 import uri from 'urijs';
+import { PublicKey } from '../data/KeyPair';
```
Comment (MichaelKim20) [inline src/modules/net/BOAClient.ts:26]: I'd like you to put it up on top of the blank line.

File: src/modules/net/response/UnspentTxOutput.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+
+    Contains definition for the unspentTxOutput
```
Comment (MichaelKim20) [inline src/modules/net/response/UnspentTxOutput.ts:3]: How about this one?
```suggestion
    Contains definition for the unspent transaction output
```

File: tests/BOAClient.test.ts [hunk 3]
```diff
@@ -120,6 +134,21 @@ function LocalNetworkTest(port: string, test: (onDone: () => void) => void)
         if (!done) res.status(204).send();
     });
 
+    // http://localhost/utxo
+    test_app.get("/utxo/:address",
+        (req : express.Request , res : express.Response) =>
+    {
+        let address: boasdk.PublicKey = new boasdk.PublicKey(req.params.address);
+
+        if (sample_utxo_address == address.toString())
```
Comment (MichaelKim20) [inline tests/BOAClient.test.ts:143]: It's very good.

File: src/modules/net/response/UnspentTxOutput.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import { Hash }  from '../../data/Hash';
+import { Height } from '../../data/Height';
```
Comment (MichaelKim20) [inline src/modules/net/response/UnspentTxOutput.ts:15]: This is an unused import
```suggestion
```

**Review by MichaelKim20 — APPROVED** [review]
It's very good. Except for a few simple styles.

**Comment by TrustHenry** [comment]
@MichaelKim20 Thanks for the reviews. I fixed it.

**Pull request merged**